include/natalie/file_stat_object.hpp

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "natalie/forward.hpp"
+#include "natalie/object.hpp"
+#include <filesystem>
+
+namespace Natalie {
+
+class FileStatObject : public Object {
+public:
+    FileStatObject(const StringObject &other)
```

Comment:
This argument is unused, yes?

Reply:
Unsure how to point to a line of code, but there is a comment below showing where the argument gets used in file_object.cpp line 409.  While it's stubbed out right now and the other argument goes unused (probably should be renamed from `other` to `path`, actually)... I do intend to use it it in the future for calls from `File.stat`, and `File#stat`, etc as opposed to `File::Stat.new`